### DISCUSSION TITLE
getting time format error

### DISCUSSION THREAD

**Discussion by ghost** [dtcenter/METplus#2387] [Configuration] [answered]
Hi,
I am trying to run pointstat but getting KeyError time format issue while running. 
Running script here `hera:/scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/RRFS/run_metplus_Point_Glb.sh` and getting error like : 
```
Fatal error occurred
KeyError: 'valid?fmt=%Y%m%d'
```
Any help on this.
Thanks

**Comment by georgemccabe** [comment, marked as answer]
Hi @perthsb,

From the log file in your output directory, I see that the error occurs when trying to read `FCST_POINT_STAT_INPUT_DIR`. I checked the final conf file that was generated from the run and saw the following values:

```
FCST_POINT_STAT_INPUT_DIR = /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/RRFS/rrfs.{valid?fmt=%Y%m%d}/00
FCST_POINT_STAT_INPUT_TEMPLATE = rrfs.t00z.prslev.f{lead?fmt=%3H}.grib2
```

You are using METplus version 5.0.1, which does not allow filename template tags like `{valid?fmt=%Y%m%d}` in config variables ending with `_DIR`. These tags must all go into the corresponding `_TEMPLATE` variable. Starting in version 5.1.0, filename template tags are allowed in `_DIR` variables.

To fix this for v5.0.1, update the above variables so that all of the filename template tags are found in the `_TEMPLATE` variable:

```
FCST_POINT_STAT_INPUT_DIR = /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/RRFS
FCST_POINT_STAT_INPUT_TEMPLATE = rrfs.{valid?fmt=%Y%m%d}/00/rrfs.t00z.prslev.f{lead?fmt=%3H}.grib2
```

**Reply by georgemccabe** [reply]
Hi @perthsb, did you resolve this issue? Let me know if you need additional assistance.
